db/app.controller.js

```diff
@@ -19,7 +18,7 @@ return fs.readFile(`./endpoints.json`,'utf-8').then(result =>{
 
```

Comment:
As you're requiring in your JSON object, you wouldn't have to read your file, you can just send the endpoints back directly